package/MDAnalysis/topology/LAMMPSParser.py

```diff
@@ -166,6 +177,65 @@
     'xy xz yz',
 ])
 
+# List of valid atom styles for Atoms section:
+# Each atom style can optionally have three image flags: nx, ny, nz
+# This mapping is used between LAMMPS and MDAnalysis attributes:
+# LAMMPS -> MDAnalysis: atom-ID -> id, molecule-ID -> segid, atom-type -> type
+# q -> charge.
+# Some commments about the some atom styles:
+# 'smd': molecule is defined differently from molecule-ID.
+# 'tdpd': this style can have n+5 columns for n cc species.
+# 'template': the order of id and resid is swapped.
+# 'hybrid': this style can have n+5 columns for n sub-styles.
+# See below for an up-to-date list of valid atom styles and their usage:
+# https://docs.lammps.org/atom_style.html
+# See below for an up-to-date list of valid atom styles and how they defined in
+# a LAMMPS DATA file:
+# https://docs.lammps.org/read_data.html
+
+# NOTE: In MDAnalysis, 'id', 'type', 'x', 'y', 'z' are the mandatory
+# attributes, and 'segid' and 'charge' are the optional ones.
+# What about the other attributes defined in LAMMPS and cannot/are not
+# mapped to corresponding attributes in MDAnalysis? Are they ignored as
+# mentioned in the doc above?
+# An idea is to keep all the atom styles defined below, and just read in
+# the attributes that are compatible with MDAnalysis and ignore the rest.
+# This way, MDAnalysis can read all types of LAMMPS data file but only read
+# in the 5 mendatory and 2 optional attributes, and warns about how it handles
+# atom styles that have other attributes than these 7 attributes.
+
+ATOM_STYLES = {
```

Comment:
Per LAMMPS documentation(see [read_data](https://docs.lammps.org/read_data.html)), the line formats for each [atom_style](https://docs.lammps.org/atom_style.html) can optionally have 3 flags (nx,ny,nz) appended to it, which indicate which image of a periodic simulation box the atom is in. In this implementation, these three additional columns of data are not parsed